bitnami/mongodb/values-production.yaml

```diff
@@ -533,7 +630,7 @@ externalAccess:
 arbiter:
   ## Enable deploying the MongoDB Arbiter
   ##   https://docs.mongodb.com/manual/tutorial/add-replica-set-arbiter/
-  enabled: true
+  enabled: false
```

Comment:
Thanks for the tip. So looking through the bitnami/moodle chart, which part should I be adding to my changes?
Is it the ingress portion (## Configure the ingress resource that allows you to access theMoodle installation) utilizing Cert Manager? If so can you explain to me how that works.

Reply:
I was thinking about the `certificates` parameter in [values.yaml](https://github.com/bitnami/charts/blob/a7deec3b503263f58d15b510d477d231dd14b066/bitnami/moodle/values.yaml#L523). If I understood it correctly, that implementation is similar to what you want to achieve.

Reply:
@FraPazGal hmm it may be...I pulled down the chart in an attempt to run it with those certificate parameters but the README is not helpful as to what exactly it needs in order for this initContainer to create and generate the certs.

Reply:
Hi!

A was taking a closer look at the `certificates` implementation on the moodle chart and it seems it is not configured correctly. I have created an internal task to fix that and a few other charts that use the same implementation.

In the meantime, you could check this PR #2970. In `certificates.customCertificate` you can add the final certificates you want to install in the system and in `certificates.customCAs` your trusted rootCAs.

Reply:
Hello. I'll take a look at it and see. This still may be a little different than what we want. Mongo's cert process is a little different overall because for TLS, they require PEM certs for each instance. Therefore the CA would need to generate certs on the fly for each replicaset that gets spun up. So when a mongo chart gets deployed with **replicas:4**, each would be spun up with their own signed cert from the CA and be located on the same directory in each pod. This way the client can connect to whichever replicaset using the signed cert that was generated for it.
For this instance the CA would be created either automatically or by the users customCA. Which ever method gets chosen, that CA should be able to generate signed PEM certs for each replicaset.